pkg/timelock/timelock.go

```diff
@@ -64,6 +73,10 @@ func NewTimelockWorker(nodeURL, timelockAddress, callProxyAddress, privateKey st
 		return nil, fmt.Errorf("poll-period must be a positive non-zero integer: got %d", pollPeriod)
 	}
 
+	if slices.Contains(httpSchemes, u.Scheme) && listenerPollPeriod <= 0 {
```

Comment:
With very low poll values, is very possible that we encounter rate limit issues. I don't know if we should enforce something in code, but definitely let users know in docs

Reply:
Good point. I'll add a note to add something about it in a follow up "doc" PR.